openedx_events/learning/signals.py

```diff
@@ -136,3 +137,14 @@
         "configuration": CourseDiscussionConfigurationData
     }
 )
+
+# .. event_type: org.openedx.learning.course.persistent_grade.summary.v1
+# .. event_name: PERSISTENT_GRADE_SUMMARY
+# .. event_description: emitted when a grade changes in the course
+# .. event_data: GradeData
+PERSISTENT_GRADE_SUMMARY = OpenEdxPublicSignal(
```

Comment:
According to the naming convention of the previous signals we're trying to maintain, this event should be called:

```
event_type: org.openedx.learning.course.persistent_grade_summary.changed.v1
PERSISTENT_GRADE_SUMMARY_CHANGED
```